gst/tensor_merge/gsttensormerge.c

```diff
@@ -257,6 +257,12 @@ gst_tensor_merge_request_new_pad (GstElement * element, GstPadTemplate * templ,
 
   tensor_merge = GST_TENSOR_MERGE (element);
 
+  if (tensor_merge->tensors_config.info.num_tensors >= NNS_TENSOR_SIZE_LIMIT) {
```

Comment:
Is it correct to use ```>=``` here, not ```>``` ?

Reply:
If num_tensors value is 16, then this request_new_pad callback will try to make sink_16 pad and num_tensors will be 17.